pkgs/games/dwarf-fortress/dwarf-therapist/default.nix

```diff
@@ -6,17 +6,23 @@
 , cmake
 , texlive
 , ninja
+, isV50 ? true
 }:
 
 stdenv.mkDerivation rec {
   pname = "dwarf-therapist";
-  version = "41.2.2";
+
+  # 41.2.5 is the last version to support Dwarf Fortress 0.47.
+  version = if isV50 then "42.1.5" else "41.2.5";
```

Comment:
Not a typo.

Reply:
This seems really hacky. Can't we use an override?